### DISCUSSION TITLE
Google is actively fighting with domains containing "adguard" word unfairly marking them as dangerous sites.

### DISCUSSION THREAD

**Discussion by dzikus** [AdguardTeam/AdGuardHome#7052] [General]
My private AdGuard panel is exposed to Internet by "adguard.mydomain.tld" and secured by Authelia 2fa + Cloudflare.

In Februrary Google Chrome started flagging my site as dangerous without any reason (any content isn't served without 2fa). I added my domain to Google Search Console and opened ticket about that - a few days and dangerous site mark dissapered without any comment or response from Google.
Then in April my site again was marked as dangerous - again I opened ticket to Google again and dangerous site mark dissapered second time without any comment from Google.
Now (June) my site is again marked as dangerous and Google Search Console claim that it isn't so I cannot open any more ticket about this problem. A month ago I started to serving AdGaurd panel also from other domain "jebacgoogle.mydomain.tld" and Google doesnt mark it as dangerous (same site, same content, authelia 2fa, only the domain is different).

Conclusion: Google is actively fighting with domains containing "adguard" word unfairly marking them as "dangerous sites trying to steal your password" without a reson to do that other then "adguard" word as subdomain.

Does anyone have a similar problem?

PS. I also serve many other subdomains from the same infrastructure and the same domain, secured in exactly the same way with exactly the same Authelia 2fa + Cloudflare - never marked by Google as dangerous sites trying to steal your passwords as Google Chrome claims with the domain "adguard.mydomain.tld".

**Comment by dzikus** [comment]
![image](https://github.com/AdguardTeam/AdGuardHome/assets/3209066/75e1a42b-9832-4d54-a8e8-027ce4ef94a5)

**Reply by snapsl** [reply]
Is your adguard publicly available? Is it used for DNS amplification attacks? Then it's good that Google marks your site as dangerous.

**Reply by dzikus** [reply]
It's not available. Even the web management interface I'm talking about here is hidden by CloudFlare+Authelia and doesn't make even a single byte of information publicly available.
